app/(route)/playground/[uid]/_nav.tsx

```
@@ -0,0 +1,260 @@
+  return (
+    <div className="flex flex-row items-center gap-[10px]">
+      <div className="text-[16px] font-bold text-[#F5F5F5]">AUTOSAVE</div>
+      <div
+        onClick={handleAutoSave}
+        onKeyDown={(e) => {
+          if (e.key === 'Enter') {
+            handleAutoSave();
+          }
+        }}
+        tabIndex={0}
+        role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)